extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.instant.js

```diff
@@ -84,13 +88,11 @@ require([
 				comscore: !win.wgDevelEnvironment
 			},
 			autoplay: willAutoplay,
-			captions: {
-				enabled: featuredVideoCookieService.areCaptionsEnabled()
-			},
+			selectedCaptionsLanguage: featuredVideoCookieService.getCaptions(),
```

Comment:
all our jwplayer properties are single-word `tracking`, `autoplay`, `settings`, `mute` etc. How about naming this property `captions`?

Reply:
afair, "captions" were wrongly interpreted as "boolean"